Makefile

```diff
@@ -27,3 +27,5 @@ add-dep:
 lint:
 	golangci-lint run
 
+run:
+	go run ./cmd/baton-github
```

Comment:
Why add this? People can just run the binary or `go run ...` right? And then they wouldn't have to deal with annoyances like `make` failing to pass command line arguments along.